modules/apigee_m10n_add_credit/src/AddCreditService.php

```diff
@@ -503,4 +525,61 @@ public static function inlineEntityFormTableFieldsPriceCallback(ProductVariation
     return t('N/A');
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function subscriptionFormAlter(&$form, FormStateInterface $form_state, $form_id) {
```

Comment:
I'm wondering if this should be a part of the `apigee_m10n` module. Some monetization dev portals may not be using the `apigee_m10n_add_credit` module because they are using recurly or some other custom solution. In that case they would loose validation.

Reply:
I've moved the logic to the main _apigee_m10n_ module. The configuration option to disable the workflow is now on the "Prepaid balance" config page (_/admin/config/apigee-edge/monetization/prepaid-balance_).
![Screenshot_2019-04-26 Prepaid balance](https://user-images.githubusercontent.com/4062676/56841469-cdb78f00-6842-11e9-94a8-29a3e1d665a0.png)

Reply:
If a developer is Prepaid, insufficient balance checks should never be done since they should not see a prepaid balance tab.